oshmem/shmem/c/profile/defines.h

```diff
@@ -102,6 +111,16 @@
 #define shmem_ulong_p                pshmem_ulong_p
 #define shmem_ulonglong_p            pshmem_ulonglong_p
 #define shmem_longdouble_p           pshmem_longdouble_p
+#define shmem_int8_p                 pshmem_int8_p
+#define shmem_int16_p                pshmem_int16_p
+#define shmem_int32_p                pshmem_int32_p
+#define shmem_int64_p                pshmem_int64_p
+#define shmem_uint8_p                pshmem_uint8_p
+#define shmem_uint16_p               pshmem_uint16_p
+#define shmem_uint32_p               pshmem_uint32_p
+#define shmem_uint64_p               pshmem_uint64_p
+#define shmem_size_p                 pshmem_size_p
+#define shmem_ptrdiff_p              pshmem_ptrdiff_p
 
 #define shmemx_int16_p               pshmemx_int16_p
 #define shmemx_int32_p               pshmemx_int32_p
```

Comment:
@hoopoepg Do you still need this ?

Reply:
Actually not, but backward compatibility...